website/docs/hoodie_cleaner.md

```diff
@@ -1,44 +1,82 @@
 ---
 title: Cleaning
 toc: true
+toc_min_heading_level: 2
+toc_max_heading_level: 4
 ---
+## Background
+Cleaning is a table service employed by Hudi to reclaim space occupied by older versions of data and keep storage costs 
```

Comment:
lets add 
hoodie.clean.max.commits to this doc

Reply:
done.